src/shared-components/button/components/linkButton/index.tsx

```diff
@@ -66,8 +66,10 @@ export const LinkButton = ({
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...rest}
     >
-      <ButtonContents hasIcon={false}>
-        <ButtonText>{children}</ButtonText>
+      <ButtonContents isLoading={false} isFullWidth={false} hasIcon={false}>
```

Comment:
Rather than rely on casting undefined props in `style.ts`, we explicitly pass in what is the default behavior we've expressed. This makes it clearer from the markup outside of `style.ts` what kinds of props the component expresses.